sentry-android-ndk/src/main/java/io/sentry/android/ndk/IModuleListLoader.java

```diff
@@ -0,0 +1,10 @@
+package io.sentry.android.ndk;
+
+import io.sentry.protocol.DebugImage;
+
+/** Used only for making the Module list loader testable */
+interface IModuleListLoader {
+  DebugImage[] getModuleList();
```

Comment:
`l`: Why not change this to
```suggestion
  List<DebugImage> getModuleList();
```
when `DebugImagesLoader` converts this to a list anyway? We could also return an empty list if native returns `null`.

Reply:
this is the replication of the Android NDK method, it only supports `jobjectArray` but not a List type